src/injection/react-native-web.js

```diff
@@ -10,7 +10,7 @@ const {
   Dimensions,
   Easing,
 } = require('react-native-web');
-const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry');
+const StyleRegistry = require('react-native-web/dist/apis/StyleSheet/registry').default;
```

Comment:
if this is the case, that's a bug in this endpoint. Babel interop should NEVER be exposed to consumers.

Please file a bug on react-native-web; nothing in their `dist` folder should have a `.default`, full stop.